### ISSUE TITLE
Mongol 18A9 is anchored badly

### ISSUE THREAD

**Issue by Mercury13** [notofonts/mongolian#4]

**Comment by simoncozens** [comment]
This is true, and should be fixed, but requires adding anchors to all the base glyphs. We will do it, but later.

**Comment by Mercury13** [comment]
I remade the font for my Unicodia, and can assure that simple anchoring will help.

**Comment by Mercury13** [comment]
As you know, I use FontForge, so I need to do the same in other software. I anchored just 25CC for now.

**Comment by Mercury13** [comment]
I tried to do it in FontLab, and changes were not so local.

**Comment by Mercury13** [comment]
What software should I use?

**Comment by simoncozens** [comment]
Fixed in #33. Will be released this quarter.
